src/backend/distributed/shared_library_init.c

```diff
@@ -886,6 +886,18 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.distribute_local_views",
+		gettext_noop(
+			"Enables distribution of local views. It is intended to be used only before "
```

Comment:
as per the above comment, we should change the comment for this GUC, this should be usable by any user, not specific to vanilla tests